#*H
#  Copyright 2017 Transport Systems Catapult (The Catapult)
#  All rights reserved
#  For use only for the purpose agreed with The Catapult
#H*

string node
int16 major_version
int16 minor_version
int16 build_version
bool valid
string versionString
uint32 localChanges
uint32 additionalCommits
